Build fails ...

After running `npm install` then running the build ...

```
C:\projects\OpenFinD3FC>grunt build
Running "showcase" task
Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module 'time-grunt'
Warning: Task "build" not found. Use --force to continue.

Aborted due to warnings.

Running "copy:main" (copy) task


Running "connect:livereload" (connect) task
Started connect web server on http://0.0.0.0:5000

Done, without errors.
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,8 @@
   "name": "OpenFinD3FC",
   "version": "0.0.0",
   "scripts": {
-    "test": "grunt ci"
+    "test": "grunt ci",
+    "postinstall": "cd node_modules/d3fc-showcase && npm i"
   },
   "devDependencies": {
     "connect-livereload": "^0.4.0",

